kubernetes/windows/Dockerfile

```diff
@@ -56,6 +56,7 @@ COPY ./amalogswindows/out_oms.so /opt/amalogswindows/out_oms.so
 
 # copy fluent, fluent-bit and out_oms conf files
 COPY ./amalogswindows/installer/conf/fluent.conf /etc/fluent/
+COPY ./amalogswindows/installer/conf/fluent-win-ama.conf /etc/fluent/

```

Comment:
nit: win in the name is redundant since the file is already present under build/windows/installer